client/src/app/site/agenda/components/agenda-list/agenda-list.component.html

```diff
@@ -202,6 +208,16 @@
                 <span translate>Edit details</span>
             </button>
 
+            <!-- Duplicate button -->
+            <button
+                *ngIf="item.contentObjectData.collection === 'topics/topic'"
```

Comment:
```suggestion
                *ngIf="item.contentObjectData.collection === Topic.COLLECTIONSTRING"
```

Reply:
That is not possible, since `Topic` is undefined in the template.